CrewChiefV4/Audio/PlaybackModerator.cs

```diff
@@ -216,10 +303,44 @@ public static bool ShouldPlaySound(SingleSound singleSound, SoundMetadata soundM
                     return false;
                 }
             }
-
             return true;
         }
 
+        public static bool MessageCanBeQueued(QueuedMessage queuedMessage, int currentQueueDepth, DateTime now)
+        {
+            int priority;
+            SoundType type;
+            if (queuedMessage.metadata == null)
+            {
+                priority = SoundMetadata.DEFAULT_PRIORITY;
```

Comment:
is it common for message to have no metadata?  If no, maybe debug text will be handy.

Reply:
not really, will add logging